cmd/rancher-desktop-guestagent/main.go

```diff
@@ -150,8 +152,16 @@ func main() {
 
 		if *enableKubernetes {
 			group.Go(func() error {
+				k8sServiceListenerIP := net.ParseIP(*k8sServiceListenerAddr)
+
+				if k8sServiceListenerIP == nil || !(k8sServiceListenerIP.Equal(net.IPv4zero) ||
+					k8sServiceListenerIP.Equal(net.IPv4(127, 0, 0, 1))) { //nolint:gomnd
+					log.Fatalf("empty or none valid input for Kubernetes service listener IP address %s. "+
```

Comment:
I don't think the `+` sign is needed you can just do the formating like:
```
log.Fatalf("empty or none valid input for Kubernetes service listener IP address %s. Valid options are 0.0.0.0 and 127.0.0.1.", *k8sServiceListenerAddr)
```

Reply:
I'm not completely sure, if I understand it correctly. Do you suggest to put the whole statement in one line?

I added the `+`/line break because otherwise the line gets pretty long (173 chars).